core/trino-main/src/main/java/io/trino/execution/DropMaterializedViewTask.java

```diff
@@ -54,9 +56,23 @@ public String getName()
         Session session = stateMachine.getSession();
         QualifiedObjectName name = createQualifiedObjectName(session, statement, statement.getName());
 
-        Optional<ConnectorMaterializedViewDefinition> view = metadata.getMaterializedView(session, name);
-        if (view.isEmpty()) {
+        Optional<ConnectorMaterializedViewDefinition> materializedView = metadata.getMaterializedView(session, name);
+        if (materializedView.isEmpty()) {
             if (!statement.isExists()) {
+                Optional<ConnectorViewDefinition> view = metadata.getView(session, name);
+                if (view.isPresent()) {
+                    throw semanticException(
+                            TABLE_NOT_FOUND,
+                            statement,
+                            "Materialized view '%s' does not exist. Attempting to use materialized view DDL syntax on a view. Did you mean DROP VIEW %s?", name, name);
```

Comment:
"DDL" is not a very intuitive term for users. I would just rephrase as:

> Materialized view '%s' does not exist, but a view with that name exists. Did you mean DROP VIEW %s?

Reply:
ok, I updated all the new error messages to this pattern.